c2corg_ui/static/js/trackdownload.js

```diff
@@ -66,7 +66,18 @@ app.TrackDownloadController.prototype.downloadFeatures_ = function(
     var content = format.writeFeatures(features, {
       featureProjection: 'EPSG:3857'
     });
-    this.download_(content, filename, mimetype + ';charset=utf-8');
+
+    // Safari does not properly work with FileSaver. Using the the type 'text/plain'
+    // makes it a least possible to show the file content so that users can
+    // do a manual download with "Save as".
+    // https://github.com/eligrey/FileSaver.js/issues/12
+    // TODO delete after Safari will be fixed
+    if (navigator.userAgent.indexOf('Safari') != -1 && navigator.userAgent.indexOf('Chrome') == -1) {
```

Comment:
This code seems a bit inoptimal: why duplicating the ``this.download_`` line when you only change the mime type?
What about simply
```
if (navigator.userAgent.indexOf('Safari') != -1 && navigator.userAgent.indexOf('Chrome') == -1) {
  mime_type = 'text/plain';
}
```
?